composer.json

```diff
@@ -10,11 +10,16 @@
         {"name": "Steve Müller", "email": "st.mueller@dzh-online.de"}
     ],
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "squizlabs/php_codesniffer": "~1.0"
```

Comment:
@gianarb It was just copied from line bellow: https://github.com/doctrine/coding-standard/pull/4/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L16

For bumping to PHP_CodeSniffer 2.0 I'd recommend standalone PR. There are many BC breaks.